app/assets/javascripts/notification.js.coffee

```diff
@@ -6,7 +6,20 @@ $(document).on 'click', '[data-behavior~=update-is_read]', (_this) ->
       is_read: @checked
       id: @id
   notification = document.getElementById('notification' + @id)
+  check_box = document.getElementById(@id)
+  unread_link = document.getElementById("unread")
+
+  #Parse number of unread notifications
+  count = 0
+  if (/[0-9]/.exec(unread_link.text) != null)
+    count = parseInt(/[0-9]+/.exec(unread_link.text))
```

Comment:
I think that it is possible to create .js script, that executes after main page rendering, which will initialize number of unread notification. I am working on it